pkgs/os-specific/linux/hwdata/default.nix

```diff
@@ -20,7 +20,7 @@ stdenv.mkDerivation rec {
   doCheck = false; # this does build machine-specific checks (e.g. enumerates PCI bus)
 
   outputHashMode = "recursive";
```

Comment:
This was in https://github.com/iblech/nixpkgs/commit/53df81926a769c262d39d36186e0fea28cc8ec6c turned into a FOD and I can't imagine why we need this here.


```suggestion
```